src/models/AmountMoney.ts

```diff
@@ -0,0 +1,7 @@
+interface AmountMoney {
+    amount?: number;
```

Comment:
`amount` should be required [per documentation](https://github.com/square/react-native-square-reader-sdk/blob/master/docs/reference.md#money).